kalsmic/iReporterApi
# E4218360 integrate delete specific

**This pull request implements delete a specific intervention or red-flag record functionality:-**
Logic implemented
- Admin user cannot delete a record
- Only  the user who created the record can delete it
- Only records whose status is draft can be deleted

**user story** https://www.pivotaltracker.com/epic/show/4218360

**How to test**
- log in  as an admin user and view details of a red-flag or interventions, you will notice there is no delete incident button
- log in as a normal user and view details of a red-flag or interventions. Try editing the deleting a record whose status is draft
- You will also notice that the delete button is hidden for records whose status is not draft.

**Links**
- UI :  https://kalsmic.github.io/iReporterApi/UI/index.html

**Credentials**
- Username: `arthur2` Password: `Password123`
- Usernmae: `admin` Password: `iReporter123`

@NicholusMuwonge @Rhytah @JamesMudidi @sanya-kenneth @frankopkusianwar @AtamaZack @ManuelDominic @KabohaJeanMark @nadralia @reifred @llwasampijja @richien @Genza999 @3Nakajugo @e-ian @Manorlds-Eaglespark @PatrickMugayaJoel @bisonlou @v1b3m @mwinel @sebbss, @kbjude
